common/djangoapps/third_party_auth/pipeline.py

```diff
@@ -437,35 +448,11 @@ def parse_query_params(strategy, response, *args, **kwargs):
     # given time. If something changes this convention in the future, please look
     # at the `login_analytics` function in this file as well to ensure logging
     # is still done properly
-    return {
-        # Whether the auth pipeline entered from /dashboard.
-        'is_dashboard': auth_entry == AUTH_ENTRY_DASHBOARD,
-        # Whether the auth pipeline entered from /login.
-        'is_login': auth_entry in [AUTH_ENTRY_LOGIN, AUTH_ENTRY_LOGIN_2],
-        # Whether the auth pipeline entered from /register.
-        'is_register': auth_entry in [AUTH_ENTRY_REGISTER, AUTH_ENTRY_REGISTER_2],
-        # Whether the auth pipeline entered from an API
-        'is_api': auth_entry == AUTH_ENTRY_API,
-    }
+    return {'auth_entry': auth_entry}
```

Comment:
Yes.  The `login_analytics` method was the last thing on my TODO list to investigate.  It wasn't updated in our previous PR when the `AUTH_ENTRY_LOGIN_API` was introduced.  Both the above comment and the `login_analytics` method should be updated.